src/metabase/sync/analyze/classifiers/category.clj

```diff
@@ -14,17 +14,33 @@
   (or (isa? base-type :type/DateTime)
       (isa? base-type :type/Collection)))
 
+(defn- maybe-make-field-a-category [distinct-count field]
+  (when (< distinct-count field-values/category-cardinality-threshold)
+    (log/debug (format "%s has %d distinct values. Since that is less than %d, we're marking it as a category."
+                       (sync-util/name-for-logging field)
+                       distinct-count
+                       field-values/category-cardinality-threshold))
+    (assoc field
+      :special_type :type/Category
+      :has_field_values "list")))
+
+;; TODO - huh? What's the difference between the two?
+(defn- maybe-make-field-a-list [distinct-count field]
+  (when (< distinct-count field-values/list-cardinality-threshold)
+    (log/debug (format "%s has %d distinct values. Since that is less than %d, we're marking it as a 'list' category." ; TODO - (?)
+                       (sync-util/name-for-logging field)
+                       distinct-count
+                       field-values/list-cardinality-threshold))
+    (assoc field
+      :special_type :type/Category
```

Comment:
we should remove the "catgeory" from something just because it has a list.
"category" should probably be renamed "low cardinality field" or something similar. shorthand would be "can we break this out on the x-axis on a bar chart", if so, it's a category.

Reply:
right